test/unit/command/init/test_template.py

```diff
@@ -29,7 +29,7 @@
 def _command_args():
     return {
         'role_name': 'test-role',
-        'url': 'https://github.com/retr0h/cookiecutter-molecule.git',
+        'url': 'https://github.com/ansible/cookiecutter-molecule.git',
```

Comment:
molecule-cookiecutter

Reply:
Thanks!

I would have expected the tests to catch this, but I'll fix it nonetheless. :)

`test/functional/docker/test_scenarios.py::test_command_init_role_with_template PASSED`

/me scratches chin...

Reply:
Links updated.